.npmignore

```diff
@@ -0,0 +1,4 @@
+.github
+.eslintrc
+.eslintignore
```

Comment:
omitting these will cause `npm test` to fail

Reply:
It doesn't look like listing is included in the tests

Reply:
`npm test` runs `npm run pretest` which runs `eslint`.